CustomerIODataPipelines.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |spec|
   spec.name         = "CustomerIODataPipelines"
-  spec.version      = "2.10.1" # Don't modify this line - it's automatically updated
+  spec.version      = "2.10.2" # Don't modify this line - it's automatically updated
```

Comment:
Isn't this auto updated? 🤔

Reply:
it would have been if this was part of `main` but unless its merged in main, it doesn't get updated with new version being rolled out.